apps/dotcms-ui/src/app/portlets/dot-templates/dot-template-create-edit/store/dot-template.store.ts

```diff
@@ -271,6 +315,39 @@ export class DotTemplateStore extends ComponentStore<DotTemplateState> {
         return result;
     }
 
+    /**
+     * When we save the properties, we do not want to save the body/layout.
+     * Therefore, we keep the same body/layout of the working template
+     */
+    private updateDraftTemplateProperties(
+        currentTemplate: DotTemplateItem,
+        template: DotTemplateItem
```

Comment:
what's the diff between template and currentTemplate? we might need better naming here.